sigsci-nginxinc-values.yaml

```diff
@@ -0,0 +1,38 @@
+controller:
+
+  image:
+    ## Replaces the default nginx-controller image with a custom image that contains the Signal Sciences nginx Module
+    repository: signalsciences/sigsci-nginxinc-nginx-ingress-controller
```

Comment:
(reminder to update this if you remove the second `nginx` from above)

Reply:
done